Assets/Scripts/GUI/OptionButton.cs

```diff
@@ -26,13 +27,40 @@ public class OptionButton : BaseButton
         [SerializeField] protected bool m_CenterPopupOnButton = false;
         [SerializeField] protected Vector3 m_PopupOffset;
         [SerializeField] protected string m_PopupText = "";
+        [SerializeField] protected LocalizedString m_LocalizedPopup;
         [SerializeField] protected string m_ToggleOnDescription = "";
+        [SerializeField] protected LocalizedString m_LocalizedToggleOnDescription;
         [SerializeField] protected Texture2D m_ToggleOnTexture;
         [SerializeField] protected bool m_AllowUnavailable = false;
         [SerializeField] private GameObject m_LinkedUIObject;
         protected string m_DefaultDescription;
         protected Texture2D m_DefaultTexture;
 
+        public string ToggleOnDescription
+        {
+            get
+            {
+                if (m_LocalizedToggleOnDescription.TableReference != null)
+                {
+                    return m_LocalizedToggleOnDescription.GetLocalizedString();
+                }
+                return m_ToggleOnDescription;
```

Comment:
I have the same question as my above comment here - why both localized and unlocalized descriptions?